src/java/org/apache/cassandra/repair/LocalSyncTask.java

```diff
@@ -116,6 +116,7 @@ protected synchronized void startSync()
             Tracing.traceRepair(message);
 
             streamPlan = createStreamPlan();
+            logger.info("{} {} {}", previewKind.logPrefix(desc.sessionId), "Starting streaming plan with id", streamPlan.getPlanId());
```

Comment:
This line is long and doesn't fit screen when reviewed in GH, i.e., difficult to review.